config/routes.rb

```diff
@@ -1,2 +1,5 @@
 Rails.application.routes.draw do
+  root "texts#index"
+  resources :texts, :only => [:index, :show]
+  resources :movies, :only => [:index]
```

Comment:
今はこちらの書き方が普通ですね。（RuboCopなら指摘されます）

```suggestion
  resources :texts, only: [:index, :show]
  resources :movies, only: [:index]
```

Reply:
承知しました！
修正いたします。